### ISSUE TITLE
It doesn't work on Linux

### ISSUE THREAD

**Issue by afengler** [ctuffli/smart#5]
Because it does things like include bsd.prog.mk in the makefile, this can't build on Linux.

Do you have plans to make this build on linux, or have you tried building it on linux at all?

**Comment by ctuffli** [comment]
You are right; the existing Makefile is FreeBSD specific and the existing source won't compile on other operating systems. That said, the architecture does support other device models (i.e. operating system interfaces), and I have made both Windows and Linux builds of smart. Unfortunately, the library I used for Linux and Windows wasn't packaged last I checked, and creating a build of smart for Windows/Linux is involved. That said, I can make the smart sources for this available and post binaries if that helped.

**Comment by afengler** [comment]
Makes sense if it's not an easy build.  
Binaries or build instructions would be useful, I'm happy to jump through a few hoops to build something I can push out to my servers.
